apps/google-analytics-4/frontend/src/apis/api.ts

```diff
@@ -82,9 +92,9 @@ export class Api {
 
   // TODO: When we actually hook this up to the sidebar chart, we will need to update this type and schema (similar to the listAccountSummaries pattern)
   // It's currently typed like this to provide maximum flexibility until we actually integrate with the chart
-  async getRunReportData(): Promise<RunReportData> {
+  async runReports(params?: RunReportParamsType): Promise<RunReportData> {
```

Comment:
made this name change as it seemed to be a bit more succinct and follows the naming of the google endpoint itself but am for sure open to reverting the change

Reply:
I'll chime my opinion here, I think for a proxy architecture - we should just copy the endpoints name (like you did here with Google since the proxy/lambda is just literally forwarding the request for us). 

In a standard REST frontend/backend structure, we should use standard CRUD names like `create`, `get`, `update`, `delete` in our controller naming (which is not the case here).